crates/starknet-devnet-types/src/rpc/contract_class/deprecated/mod.rs

```diff
@@ -12,7 +12,13 @@ pub mod rpc_contract_class;
 
 #[derive(Debug, Clone, Eq, PartialEq)]
 pub enum Cairo0ContractClass {
-    // TODO: remove once starknet_api raised
+    RawJson(Cairo0Json),
+    Rpc(DeprecatedContractClass),
+}
+
+#[derive(Serialize, Deserialize)]
+#[serde(tag = "type", content = "contract_class")]
+enum Cairo0ContractClassSerdeHelper {
```

Comment:
What is the improvement of introducing this helper enum? And why such a tag+content scheme?